app/data/spd-mpdp.sql

```diff
@@ -0,0 +1,12 @@
+
+DROP table if exists public.schema_payment_data;
```

Comment:
I should have picked this up yesterday but could you rename the table to something like aggregate_scheme_payments, or anything similar that contains 'aggregate' in the name because the table will only contain totals as opposed to individual payments, and having it in the name makes it easier to read.

Reply:
Thanks for your comment, Done!